examples/nlp/bert_pretraining.py

```diff
@@ -60,6 +60,7 @@
 """
 import argparse
 import os
+import math
```

Comment:
please see style guide for imports
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
i dont understand whats the problem?